libnd4j/include/cblas.h

```diff
@@ -21,7 +21,7 @@
 #ifndef NATIVEOPERATIONS_CBLAS_H
 #define NATIVEOPERATIONS_CBLAS_H
 
-#include "config.h"
+// #include "config.h"
```

Comment:
This will break the build, please don't do that.

Reply:
Don't worry, it's temporary comment.